src/components/ReanimatedSwipeable.tsx

```diff
@@ -575,18 +577,23 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
             ? 'left'
             : 'right';
 
-        if (rowState.value === 0 && onSwipeableOpenStartDrag) {
-          runOnJS(onSwipeableOpenStartDrag)(direction);
-        } else if (rowState.value !== 0 && onSwipeableCloseStartDrag) {
-          runOnJS(onSwipeableCloseStartDrag)(direction);
+        if (dragStarted.value === false) {
```

Comment:
Why not

```suggestion
        if (!dragStarted.value) {
```

?

Reply:
done in [d0bda43](https://github.com/software-mansion/react-native-gesture-handler/pull/3151/commits/d0bda4353678ecdfd40a0f022b7ed742b76bbe59)